force-di/main/classes/di_Binding.cls

```diff
@@ -60,15 +60,27 @@ public abstract class di_Binding implements Comparable {
      * Gets the instance of the thing the binding points to
      **/
     public Object getInstance() {
-        return getInstance(null);
+        return getInstance(null, false);
     }
 
     /**
      * Gets the instance of the thing the binding points to
      **/
     public Object getInstance(Object params) {
+        if (params instanceOf Boolean) {
```

Comment:
So I know this looks dumb, but here is what was happening. When I had created another method with signature `public Object getInstance(Boolean newInstance)`, it was getting directed to the `public Object getInstance(Object params)`. This was the only way I could think of to perform the method overloading due to this.